src/checkpoints/checkpoints.h

```diff
@@ -131,7 +139,7 @@ namespace cryptonote
      *
      * @return a const reference to the checkpoints container
      */
-    const std::map<uint64_t, crypto::hash>& get_points() const;
+    const std::map<uint64_t, checkpoint_t>& get_points() const { return m_points; };
```

Comment:
Need to look into thread-safety implications of this.

Reply:
Yeah won't be thread-safe. I've changed it to return it by value as a temporary fix. I don't think I will end up using the map here because the number of new checkpoints is going to grow very large over time and we'll want to change to storing and retrieving from the DB so the only thread safety we would need is on the the staging checkpoints that are accumulating votes and thread safe access to the DB.